podman.spec.rpkg

```diff
@@ -240,6 +230,9 @@ fi
 %{_bindir}/%{name}
 %dir %{_libexecdir}/%{name}
 %{_libexecdir}/%{name}/rootlessport
+%{_libexecdir}/%{name}/quadlet
+%_prefix/lib/systemd/system-generators/podman-system-generator
+%_prefix/lib/systemd/user-generators/podman-user-generator
```

Comment:
these are easier:
```
%{_systemdgeneratordir}/%{name}-system-generator
%{_systemdusergeneratordir}/%{name}-user-generator
```

I don't know if rhel 8 and 9 have these defined already, let me take a quick look and get back.

Reply:
Yes, these 2 macros should work on rhel8 and 9.